about.html

```diff
@@ -0,0 +1,144 @@
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <header>
+        <div id="mobile-menu">
+            <a href="#">
+                <i class="material-icons menu">menu</i>
+            </a>
+        </div>
+        <div id="top-bar">
+            <a href="#">
+                <i class="fa-brands fa-facebook-f"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
+        </div>
+        <nav id="navigation">
+            <a id="page-logo" href="./index.html">
+                <img src="images/iCON.png" alt="Logo">
+            </a>
+            <ul id="desktop-nav">
+                <li>
+                    <a href="./index.html" class="nav-list">Home</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-list">Program</a>
+                </li>
+                <li>
+                    <a href="./about.html" id="active">About</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-list">Sponsor</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-list">News</a>
+                </li>
+                <button type="button" id="nav-btn">Join Now</button>
+            </ul>
+        </nav>
+    </header>
+    <div id="hamburger">
+        <a href="#" id="close">
+            <i id="close-icon" class="material-icons">close</i>
+        </a>
+        <ul id="hamburger-ul">
+            <li>
+                <a class="link" href="./index.html"> Home</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Program</a>
+            </li>
+            <li>
+                <a class="link" href="./about.html"> About</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Sponsor</a>
+            </li>
+            <li>
+                <a class="link" href="#"> News</a>
+            </li>
+        </ul>
+    </div>
+    <main>
+        <section id="about-intro">
+            <h3>"Hello! Data Scientists"</h3>
+            <h1>AI & BIG DATA FOR A SMARTER FUTURE EXHIBITION 2023</h1>
+            <P>
+                AI & Big Data Exhibition brings together the community of experts,
+                academics, and activists who showcase the next generation enterprise 
+                technologies and strategies from the world of Data Science and AI for 
+                a smarter future.
+            </P>
+            <div id="contact">
+                <p>Please contact us per Email for any future questions about AI & Big Data Exhibition 2023!</p>
+                <a href="#">ai-bigdata2023@rdnairobi.org</a>
+            </div>
+        </section>
```

Comment:
- [ ] Good job so far your design is good. But your about.html page header section looks a bit different from the original design. Please make sure that every part of your header section looks similar to the original design. As a developer it is very important to pay attention to details and stick to design as much as possible :smile: 

#### Original design
![Screenshot from 2022-12-02 18-04-44](https://user-images.githubusercontent.com/6615394/205357532-b4e8ef5e-525d-436d-8a03-5ef6252ee89e.png)

#### Your design
![Screenshot from 2022-12-02 18-04-14](https://user-images.githubusercontent.com/6615394/205357675-874fe394-4e8a-4f5d-895c-ce644044df7c.png)